workspaces/devurls.md

```diff
@@ -97,3 +97,13 @@ $ curl --cookie "devurl_session=<TOKEN>" <dev-url>
 
 You can also access your server via
 [SSH port forwarding](ssh.md#forwarding-dev-urls).
+
+## Troubleshooting
+
+If you're seeing issues with your dev URL, we recommend using something like
+Python's `http.server` module to gain additional information helpful for
+troubleshooting issues with DevURL configuration and external proxies:
+
+```console
+python3 -m http.server 8080
```

Comment:
do we want to add a note that `8080` should be whatever port the dev URL points to?

Reply:
👍  It can't hurt to mention that!